internal/backends/winit/renderer/femtovg/glcontext.rs

```diff
@@ -121,18 +122,23 @@ impl OpenGLContext {
             })?
             .map(|h| h.as_raw());
 
-        let gles_context_attributes = ContextAttributesBuilder::new()
-            .with_context_api(ContextApi::Gles(Some(glutin::context::Version {
-                major: 2,
-                minor: 0,
-            })))
-            .build(raw_window_handle);
+        let preferred_context_attributes = match opengl_api {
+            Some(OpenGLApi::GL) => ContextAttributesBuilder::new()
+                .with_context_api(ContextApi::OpenGl(None))
+                .build(raw_window_handle),
+            Some(OpenGLApi::GLES) | None => ContextAttributesBuilder::new()
+                .with_context_api(ContextApi::Gles(Some(glutin::context::Version {
+                    major: 3,
+                    minor: 2,
```

Comment:
Since this changes the default from 2.0 to GLES 3.2, perhaps it would be better to have two GLES variants in the OpenGL API enum - or something like Glutin uses: https://docs.rs/glutin/latest/glutin/context/enum.ContextApi.html#

Reply:
I'm not sure why I updated this, possibly a testing change that made it into a commit? I intended to keep the default GLES version here, as it appears to use GLES 3.2 for me when this is set to `major: 2` anyway. I'll revert the default version change.

I do think it would be useful to be able to request a specific version, and considered adding that but wanted to get feedback first. I was thinking of the Glutin approach as well, I'll go with that.

Reply:
Pushed up a commit adding an `APIVersion` field to the `OpenGLAPI` items.